src/app/modules/group/components/group-manager-add/group-manager-add.component.html

```diff
@@ -0,0 +1,22 @@
+<alg-section-paragraph icon="fa fa-sign-in-alt" i18n-label label="Add a manager">
+  <div class="input-wrapper">
+    <input
+      #input
+      type="text"
+      i18n-placeholder placeholder="Login of a user"
+      (keyup)="0"
```

Comment:
Is this needed ? Since there is an `ngModel` two-way binding, I suppose the angular will be run the change detection and the template will be updated.